modules/oe_paragraphs_media/oe_paragraphs_media.post_update.php

```diff
@@ -31,10 +31,31 @@ function oe_paragraphs_media_post_update_00001(array &$sandbox) {
 }
 
 /**
- * Create new form modes for Text with featured Media paragraph.
+ * Rename "Title" field to "Heading".
  */
 function oe_paragraphs_media_post_update_00002(): void {
-  $storage = new FileStorage(drupal_get_path('module', 'oe_paragraphs_media') . '/config/post_updates/00002_create_form_modes');
+  $field_config = FieldConfig::load('paragraph.oe_text_feature_media.field_oe_title');
+  $field_config->setLabel('Heading');
+  $field_config->save();
+}
+
+/**
+ * Add "Title" field to the Text with featured Media paragraph.
+ */
+function oe_paragraphs_media_post_update_00003(): void {
+  // Create "Title" field storage first.
+  module_load_include('php', 'oe_paragraphs', 'oe_paragraphs.post_update');
+  oe_paragraphs_post_update_10007();
+
+  $storage = new FileStorage(drupal_get_path('module', 'oe_paragraphs_media') . '/config/post_updates/00003_add_title_field');
+  \Drupal::service('config.installer')->installOptionalConfig($storage);
+}
```

Comment:
There will be 3 post_updates in the epic regarding field updates: `oe_paragraphs_media_post_update_00001`, `oe_paragraphs_media_post_update_00002`, `oe_paragraphs_media_post_update_00003`. Why can't we merge them and then have just one post_update for the fields changes and one post_update for the form modes?

Reply:
We even can have just one post-update function. @ademarco asked to split different actions for different steps, also Link and Title field have been implemented in different tasks, so I'm trying to be in the scope of the ticket.
In conclusion - I can reorganise it in any way, let's receive someone else feedback also.

Reply:
I agree with @22Alexandra here, lets merge 00001, 00002 and 00003 since this three deal with updating fields and keep 00004 for the form mode changes.